docs/hackathons/ihp_10_2022.rst

```diff
@@ -0,0 +1,276 @@
+
+
+Hackathon at Institut Henri Poincaré, 17th-21st Oct 2022
+========================================================
+
+This hackathon happened at IHP within the thematic trimester on `Geometry and Statistics in Data Sciences <https://indico.math.cnrs.fr/category/498/>`_.
+
+
+Several participants made their first contributions to ``geomstats``. In no particular order: Francesco Ballerin (University of Bergen), Juliane Braunsmann (WWU Munster), Antoine Collas (Paris-Saclay University), John Harvey (Cardiff University), Emmanuel Hartman (Florida State University), and Joan Glaunès (Université Paris Descartes).
+
+
+Besides the usual informal schedule, we had a very fruitful presentation by Jean Feydy (INRIA), entitled "Fast geometric libraries for vision and data sciences". 
+
+
+It was a prolific week in terms of (great) discussions about math and programming. Several actions to improve our code base originated from them. The next section describes them and the rationale behind the changes that will be implemented.
+
+
+
+Future changes
+--------------
+
+
+Equip with metric
+*****************
+
+The following way of instantiating a space and equipping it with a metric will become standard:
+
+
+.. code-block:: python
+
+    from geomstats.geometry.hypersphere import (
+        Hypersphere,
+        HypersphereMetric,
+    )
+
+    space = Hypersphere(dim=2, equip=False)
+    space.equip_with_metric(HypersphereMetric)
```

Comment:
NIT: 

- In a future version,  maybe this could become space.equip(HypersphereMetric) : the space would automatically recognize that the class is a child of RiemannianMetric and perform an equip_with_metric.

Reply:
`space.equip_with` instead? I think is neat and pretty straightforward to implement.